boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-common.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54h20dk_nrf54h20-memory_map.dtsi"
+#include "nrf54h20dk_nrf54h20-ipc_conf.dtsi"
+#include "nrf54h20dk_nrf54h20-pinctrl.dtsi"
+
+&hfxo {
+	status = "okay";
+	accuracy-ppm = <30>;
+	start-up-time-us = <850>;
```

Comment:
My understanding is that such information will be read out from BICR and not fixed in DT. Is my understanding correct ?

Reply:
BICR must be generated based on data from .dts. I understand this `start-up-time-us` property should be used to generate BICR.

Then the Application CPU can use both sources because both .dts and BICR should contain the same value for the application build.
Other software components built separately from the Application must use BICR because they don't have access to the Application's .dts including HFXO properties.